lib/embed/viewer/common_viewer.rb

```diff
@@ -199,6 +199,14 @@ def file_url(title)
       end
 
       ##
+      # Creates a pretty date in a standardized sul way
+      # @param [String] date_string
+      # @return [String]
+      def sul_pretty_date(date_string)
+        Date.parse(date_string).strftime('%d-%b-%Y')
```

Comment:
Its possible but then we need to mixin ActionView::Helpers::TranslationHelper. Just one of those trade off things :/  @jkeck any opinion on that?